common/models/update_load_balancer_request.go

```diff
@@ -0,0 +1,38 @@
+package models
+
+import (
+	"fmt"
+
+	swagger "github.com/zpatrick/go-plugin-swagger"
+)
+
+type UpdateLoadBalancerRequest struct {
+	LoadBalancerID string
+	Ports          *[]Port     `json:"ports"`
+	HealthCheck    HealthCheck `json:"health_check"`
+}
+
+func (u UpdateLoadBalancerRequest) Validate() error {
+	if u.LoadBalancerID == "" {
+		return fmt.Errorf("LoadBalancerID is required")
+	}
+
+	return nil
+}
+
+func (u UpdateLoadBalancerRequest) Definition() swagger.Definition {
+	return swagger.Definition{
+		Type: "object",
+		Properties: map[string]swagger.Property{
```

Comment:
The swagger definition needs to match the fields in the `UpdateLoadBalancer` struct. To do nested objects, I believe you use `NewObjectProperty("name")`, e.g.:
```
"load_balancer_id":  swagger.NewStringProperty(),
"ports":             swagger.NewObjectSliceProperty("Port"),
"health_check":      swagger.NewObjectProperty("HealthCheck"),
```

The object names are declared and need to match in `/api/controllers/swagger.go`

Reply:
:+1: